packages/core/src/browser/style/variables-bright.useable.css

```diff
@@ -142,6 +142,12 @@ is not optimized for dense, information rich UIs.
   --theia-removed-color0: rgba(230, 0, 0, 0.8);
   --theia-modified-color0: rgba(0, 100, 150, 0.8);
 
+  --theia-search-match-color0: rgba(234, 92, 0, 0.33);
```

Comment:
Should we consider to move them to the search in workspace extension? They are not supposed to be used in core. If it is hard, please at least file a follow-up issue to make it easier to define themed variables.

Reply:
I moved them to the extension and added new variables here. Not the best solution, a bit arbitrary maybe. I guess we should edit this with https://github.com/theia-ide/theia/issues/1486. 

Regarding themed variables for extensions: we shouldn't do such thing and leave every theme related variable in core.